images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -8,7 +8,7 @@ ARG AGENTBUILD
 # NOTE: This imge is NOT based on 'githubrunners-buildpackdeps-stretch' because AzFunctions
 # team wanted a consistent experience at their end and do not want to see latency that might
 # be caused due to when the GitHub runners' layers and Oryx image layers go out of sync.
-FROM debian:stretch AS main
+FROM githubrunners-buildpackdeps-stretch AS main
```

Comment:
Changing this after discussing with jam stack team. They are seeing reports where users are unable to use certain libraries which have native dependencies and they are not present in `debian:stretch`

Reply:
it would be good to document and list out those libraries ... we will surely need this for buster images